scripts/kvt_genvk.py

```diff
@@ -21,14 +21,12 @@
 
 def startTimer(timeit):
     global startTime
-    if timeit:
-        startTime = time.process_time()
+    startTime = time.clock()
```

Comment:
This seems like an unrelated change, is it needed or should we have a custom revert commit that only changes the known-good commit

Reply:
These changes were made during the known-good update--changes in Vulkan-Headers' `genvk.py` are reflected in each repo's version of the same file when known-good is updated-- so I reverted the changes to `kvt_genvk.py` as well. I can do some investigating to see if I need to revert `kvt_genvk.py` or if I can leave it as it.

Reply:
My conclusion is the `kvt_genvk.py` changes do not need to be reverted. These updated changes have passed through Gerrit and github CI.